OTAnalytics/plugin_ui/customtkinter_gui/frame_project.py

```diff
@@ -73,6 +73,8 @@ def _get_widgets(self) -> None:
         self.frame_svz_metadata = FrameSvzMetadata(
             master=self.tab(self._svz_title), viewmodel=self._viewmodel
         )
+        if not self._viewmodel.show_svz():
+            self.delete(self._svz_title)
```

Comment:
The svz tab and all ui elements are created. Afterwards, the tab is deleted to be hidden. This was necessary, because our current approach of registering frames to the view model raises an exception if one of the frames is missing. I did not want to change that. This will be changed on the transition to a web ui.